curriculum/challenges/english/07-scientific-computing-with-python/learn-lambda-functions-by-building-an-expense-tracker/658246c28575d653d1f89b59.md

```diff
@@ -14,7 +14,7 @@ while condition:
     <code>
 ```
 
-Below the `expense` list, create a `while` loop. Use `True` for the condition, and print the string `\nExpense Tracker` inside the loop body to show the title of the program.
+Below the `expense` list, create a `while` loop. Use `True` for the condition, and print the string `'\nExpense Tracker'` inside the loop body to show the title of the program.
```

Comment:
I'm unable to check if they call the list 'expenses' but since you proposed to change to that, there are some cases you forgot to rename it like here

Reply:
I'm not renaming, the list is called `expenses` and this was a typo